src/static/css/ebook.css

```diff
@@ -0,0 +1,252 @@
+#title img {
+  margin: 0 auto;
+}
+
+.byline {
+    font-style: italic;
+}
+
+.byline.authors {
+    font-weight: bold;
+    border-bottom: none;
+}
+
+.table-wrap-container {
+  margin: 0;
+}
+
+.table-wrap {
+  display: block;
+  max-width: 100%;
+  text-align: center;
+}
+
+th, td {
+  padding: 5px;
+}
+
+tbody tr:nth-child(even) {
+  background-color: #f2f2f2;
+}
+
+#toc ul {
+  list-style-type: none;
+}
+
+#toc a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.content a, .content a:visited {
+  color: #0b1423;
+}
+
+.intro-image {
+  background-color: grey;
+  border-radius: 10px;
+  display: block;
+  margin: 20px 0;
+}
+
+.team {
+  overflow: hidden;
+  height: 100px;
+  margin: 0 0 24px 0;
+}
+
+.team img {
+  height: 80px;
+  width: 80px;
+}
+
+
+.contributors {
+  padding: 0;
+}
+
+.contributor-grid {
+  margin: 0 -15px;
+  justify-content: center;
+  width: 10000px;
+  max-width: 100%;
+}
+
+.contributor {
+  margin-top: 20px;
+  width: 600px;
+  max-width: 100%;
+  vertical-align: top;
+  overflow: auto;
+}
+
+.contributor-avatar {
+  border-radius: 50px;
+  background-color: #f7f779;
+  margin-bottom: 10px;
+  margin-right: 10px;
+  width: 50px;
+  height: auto;
+  transition-duration: .3s;
+  float: left;
+}
+
+ul li.contributor::before {
+  content: "";
+}
+
+.contributor-social a {
+  display: inline-block;
+  margin: 0 5px;
+}
+
+.contributor-teams {
+  margin-top: 5px;
+}
+
+.contributor-team {
+  font-size: 14px;
+  font-size: 0.875rem;
+  color: #757575;
+}
+.contributor-team::after {
+  content: ', ';
+}
+.contributor-team:last-child::after {
+  content: '';
+}
+
+.contributors button {
+  background: #f7f779;
+  color: #333;
+}
+
+/* Join the team card */
+.contributor-join {
+  display: none;
+}
+
+.social-icon {
+  height: 20px;
+  width: 20px;
+}
+
+@page {
+  margin: 2.5cm;
+  padding: 0;
+  font-family: 'Lato', sans-serif;
+  font-size: 10pt;
+
+  @bottom-left {
+    width: 100%;
+    margin: 10pt 0 30pt 0;
+    border-top: .25pt solid #666;
+    content: "The Web Almanac 2019 by HTTP Archive";
```

Comment:
Should we have English-specific content in the base stylesheet? Or is this always overwritten by the templates?

Reply:
It's overwritten by this inline style.

https://github.com/HTTPArchive/almanac.httparchive.org/blob/f3e54ef4d4dc4f5f730f1ffaca9b7df6fb13585b/src/templates/base/2019/base_ebook.html#L19-L25

Hence why it appears as "2019 Web Almanac by HTTP Archive" in English version. Probably should update this to match. Or could remove it completely from `ebook.css` but no harm to have English fallback.

Reply:
Removing it SGTM. In 4 months if I wanted to edit this text I wouldn't remember which file to edit.

Reply:
It's gone!